### PR TITLE
Update api-plugin-parser.txt

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

File: docs/v0.14/api-plugin-parser.txt [hunk 1]
```diff
@@ -97,4 +99,76 @@ Parser plugins must implement this method.
...
+      def setup
+        # common setup
+      end
+      
+      COPNFIG = %[
+        pattern apache
+      ]
+      
+      def create_driver(conf = CONF)
+        Fluent::Test::Driver::Parser.new(Fluent::Plugin::YourOwnFormatter).configure(conf)
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-parser.txt]: `YourOwnParser`?

File: docs/v0.14/api-plugin-parser.txt [hunk 2]
```diff
@@ -97,4 +99,76 @@ Parser plugins must implement this method.
...
+            assert_equal(expected_time, time)
+            assert_equal(expected_record, record)
+          end
+        end
+      end
+    end
+
+### Overview of Tests
+
+Testing for formatter plugins are mainly for:
```
Comment (repeatedly) [inline docs/v0.14/api-plugin-parser.txt]: parser plugins

**Comment by okkez** [comment]
I've fixed names.

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**